src/test/java/jenkins/plugins/logstash/LogstashWriterTest.java

```diff
@@ -87,15 +104,15 @@ public void before() throws Exception {
     when(mockBuild.getParent()).thenReturn(mockProject);
     when(mockBuild.getBuiltOn()).thenReturn(null);
     when(mockBuild.getNumber()).thenReturn(123456);
-    when(mockBuild.getTimestamp()).thenReturn(new GregorianCalendar());
+    when(mockBuild.getTime()).thenReturn(new Date());
     when(mockBuild.getRootBuild()).thenReturn(mockBuild);
     when(mockBuild.getBuildVariables()).thenReturn(Collections.emptyMap());
     when(mockBuild.getSensitiveBuildVariables()).thenReturn(Collections.emptySet());
     when(mockBuild.getEnvironments()).thenReturn(null);
     when(mockBuild.getAction(AbstractTestResultAction.class)).thenReturn(mockTestResultAction);
-    when(mockBuild.getLog(0)).thenReturn(Arrays.asList());
+    // when(mockBuild.getLog(0)).thenReturn(Arrays.asList());
```

Comment:
please don't leave commented out code, just delete

Reply:
removed all occurrences of commented out lines that I could spot (3), thanks